### PR TITLE
Get permission for user on workspace

### PR DESCRIPTION
**Pull request by josemiguelmelo**
This endpoint intends to get the permission level for a user inside an organization.

### CODE REVIEW

**Review by ktrysmt — APPROVED** [review]
LGTM, Thank you so much!

**Comment by ktrysmt** [comment]
Please see https://github.com/ktrysmt/go-bitbucket/releases/tag/v0.8.1 .

File: tests/workspace_test.go [hunk 1]
```diff
@@ -85,3 +85,17 @@ func TestGetWorkspaceRepository(t *testing.T) {
 		t.Error("Cannot catch repos full name.")
 	}
 }
+
+func TestGetWorkspacePermissionForUser(t *testing.T) {
+	c := getBitbucketClient(t)
+	workspaceName := getWorkspace(t)
+
+	res, err := c.Workspaces.Permissions.GetUserPermissions(workspaceName, "josemiguelmelo")
```
Comment (sl4mmy) [inline tests/workspace_test.go:93]: Did you mean to leave your nickname hard-coded in this test?

Comment (ktrysmt) [reply tests/workspace_test.go:93]: Oh sorry I overlooked. @sl4mmy

**Pull request merged**